pyaedt/hfss.py

```diff
@@ -373,6 +373,14 @@ def create_frequency_sweep(self, setupname, unit="GHz", freqstart=1e-3, freqstop
         -------
         SweepHFSS
```

Comment:
```suggestion
        :class:`pyaedt.modules.SetupTemplates.SweepHFSS` or :class:`pyaedt.modules.SetupTemplates.SweepQ3D`
```
New documentation style (in next release and dev.aedtdocs.pyansys.com) will be able to directly link classes

Reply:
OK, that 's great. That's avoid any wrong spelling and in case we change the name of the class, I am assuming the link will be updated as in c#.

Reply:
> I am assuming the link will be updated as in c#

We'll have a link within the documentation, not sure what doc generator you're using for C# (doxygen?).